srcpkgs/jamulus/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'jamulus'
+pkgname=jamulus
+version=3.6.2
+revision=1
+wrksrc=Jamulus${version}
+build_style=qmake
+configure_args="Jamulus.pro"
+hostmakedepends="pkg-config"
+makedepends="qt5-declarative-devel jack-devel"
+depends="jack"
+short_desc="Play music online. With friends. For free"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://jamulus.io/"
+distfiles="${SOURCEFORGE_SITE}/project/llcon/Jamulus/${version}/Jamulus-${version}.tar.gz"
+checksum=86d38f7d22b49720a79a246918cb8f16a690bbc7d6cbc8707e93694bab1a074c
+
+if [ "${CROSS_BUILD}" ]; then
+	hostmakedepends+=" qt5-devel"
```

Comment:
Feel free to put this in `hostmakedepends` unconditionally.

Reply:
For the sake of maintaining a cross platform package, I like knowing what is needed specifically for cross builds. I'd prefer o keep it this way for the sake of maintaining, but if there's a general preference in packages not to do this, let me know and I can add `qt5-devel` unconditionally to `hostmakedepends` when I get home.

Reply:
It's ok to leave as is if you prefer it.